ipad.css

```diff
@@ -0,0 +1,374 @@
+}
+
+.box__item-1 {
+  margin-right: 85px;
+}
+
+.box__text-number {
+  font-size: 50px;
+  font-weight: 600;
+}
+
+.box__text-sec {
+  font-size: 20px;
+  margin-left: 8px;
+}
+
+.mobile-box {
+  display: flex;
+  width: 100%;
+  margin: 0 auto;
+  margin-bottom: 55px;
+}
+
+.mobile-test {
+  margin: 0 auto;
+}
+
+.test {
+  margin: 0;
+}
+
+.test__img-3 {
+  width: 30.37px;
+  height: 30.37px;
+  margin: 0;
+  margin-right: 11px;
+}
+
+.box__button {
+  font-size: 16px;
+  width: 220px;
+  height: 55px;
+}
+
+.test__text {
+  font-size: 16px;
+  display: inline-block;
+  margin: 0;
+  margin-right: 48px;
+}
+
+.main {
+  max-width: 768px;
+}
+
+.future {
+  display: block;
+  background: none;
+}
+
+.section__future-left {
+  max-width: 768px;
+  background: url(img/future.png) no-repeat;
+  background-size: cover;
+}
+
+.future__h1 {
+  display: flex;
+  justify-content: flex-end;
+  margin-left: 0;
+  margin-right: 46px;
+  height: 500px;
+  width: auto;
+  white-space: nowrap;
+}
+
+.section {
+  padding-top: 55px;
+}
+
+.section__future {
+  display: flex;
+  flex-wrap: wrap;
+  align-items: center;
+  max-width: 768px;
+}
+
+.mobile__section {
+  max-width: 318px;
+  margin: 0 82px 75px 34px;
+}
+
+.mobile__section_icon {
+  max-width: 270px;
+  margin-left: 34px;
+  padding-bottom: 50px;
+}
+
+.section__h1 {
+  margin: 0;
+  margin-bottom: 20px;
+}
+
+.section__h1_safe {
+  max-width: 316px;
+}
+
+.section__p {
+  margin: 0;
+}
+
+.section__icon {
+  margin: 0;
+}
+
+.item__icon {
+  display: flex;
+  align-items: center;
+  margin-bottom: 21px;
+}
+
+.link__section {
+  font-weight: 600;
+  font-size: 16px;
+  margin-right: 55px;
+}
+
+.link__section::before {
+  content: url(img/Group20.svg);
+  width: 31.18px;
+  height: 31.18px;
+  margin-right: 5px;
+}
+
+.section__info {
+  width: auto;
+  margin: 0 auto;
+  margin-bottom: 52px;
+}
+
+.future__button {
+  font-size: 16px;
+  width: 220px;
+  height: 55px;
+}
+
+.safe {
+  display: block;
+  position: static;
+}
+
+.section__safe {
+  width: 100%;
+  padding-bottom: 52px;
+  max-width: 100%;
+  padding-top: 60px;
+}
+
+.section__safe_img {
+  display: none;
+}
+
+.safe__img {
+  display: none;
+}
+
+.section__safe_mobile-1 {
+  max-width: 317px;
+  margin-right: 70px;
+}
+
+.section__p_safe {
+  margin-bottom: 85px;
+}
+
+.five {
+  display: none;
+}
+
+.mobile__section_flex {
+  display: flex;
+  justify-content: space-around;
+  margin-right: 70px;
+  margin-left: 34px;
+}
+
+@media screen and (max-width: 700px) {
+  .test__img-3 {
+    width: 22px;
+    height: 22px;
+    margin-right: 7px;
+    color: #e51937;
+  }
+
+  .box__button {
+    font-size: 13px;
+    width: 160px;
+    height: 40px;
+  }
+
+  .section__info {
+    width: 100%;
+  }
+
+  .mobile__section_flex {
+    flex-wrap: wrap;
+  }
+
+  .box {
+    margin: 0 auto 34px;
+    max-width: 303px;
+    flex-wrap: wrap;
+    width: 100%;
+  }
+
+  .box__img-1 {
+    width: 38.14px;
+    height: 28.7px;
+  }
+  
```

Comment:
Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/Tesla-n/pullRequest?prid=5749910)